src/sections/blog-post.liquid

```diff
@@ -0,0 +1,147 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add "Blog Posts" section on the Theme Editor.
+
+  Theme Store required settings
+  - Blog (which blog category to show in your store)
+  - Number of posts to show
```

Comment:
The heading, not listed here (but shown below as `section.settings.title`), is a required setting.